### ISSUE TITLE
Importing nodes from clipboard failed...

### ISSUE THREAD

**Issue by zhouxinghong** [LorenEteval/Furious#26]
Importing nodes from the clipboard outputs the following information:
```html
[2023-10-21 13:10:58,696] [Furious.Action.Import] [ERROR] import share link failed: ss://YWVzLTI1Ni1nY206Y2RCSURWNDJEQ3duZklOQDM4LjEyMS40My43MTo4MTE5#US_youtube%40%E8%B5%84%E6%BA%90%E5%88%86%E4%BA%AB%E5%B8%88_49. Exception: not enough values to unpack (expected 2, got 1)
[2023-10-21 13:10:58,745] [Furious.Action.Import] [ERROR] import share link failed: ss://YWVzLTI1Ni1nY206a0RXdlhZWm9UQmNHa0M0QDM4LjEyMS40My43MTo4ODgy#US_youtube%40%E8%B5%84%E6%BA%90%E5%88%86%E4%BA%AB%E5%B8%88_72. Exception: not enough values to unpack (expected 2, got 1)
```
Windows 10 x64 19044
Latest preview version

**Comment by LorenEteval** [comment]
Thank you. I added this error message in case some share links did not import successfully. This is a old SS format that Furious failed to handle. I'll fix that in next release.

By the way, Is VPN mode working properly? I didn't see you post new error logs so I guess that's solved.

**Comment by zhouxinghong** [comment]
Hello!
Judging from the current use, the stability has been greatly improved. I will continue to report to you if new errors or exceptions occur! Thank you for a really great web tool 👍👏🎉😜
I also want to ask you a question! Which country are you friends from? 😊

**Comment by LorenEteval** [comment]
> Hello! Judging from the current use, the stability has been greatly improved. I will continue to report to you if new errors or exceptions occur! Thank you for a really great web tool 👍👏🎉😜 I also want to ask you a question! Which country are you friends from? 😊

Sure, great to hear that. Then I'll close #24 as completed.

**Comment by zhouxinghong** [comment]
> > Hello! Judging from the current use, the stability has been greatly improved. I will continue to report to you if new errors or exceptions occur! Thank you for a really great web tool 👍👏🎉😜 I also want to ask you a question! Which country are you friends from? 😊
> 
> Sure, great to hear that. Then I'll close #24 as completed.

OK! no problem.
I can help you test if you have any kind of software problems in the future. 😜

**Comment by LorenEteval** [comment]
Closed as completed in [release 0.2.11](https://github.com/LorenEteval/Furious/releases/tag/0.2.11)
